src/runtime/builder.rs

```diff
@@ -68,6 +68,23 @@ impl Builder {
         self
     }
 
+    /// Create a new singlethreaded runtime
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # extern crate tokio;
+    /// # use tokio::runtime::Builder;
+    /// # pub fn main() {
+    /// let runtime = Builder::new().single_threaded().unwrap();
+    /// // ... call runtime.block_on(f) where f is a future
+    /// # let _ = runtime;
+    /// # }
+    /// ```
+    pub fn single_threaded(&mut self) -> io::Result<SingleThreaded> {
```

Comment:
I'm wondering if this should be named `build_single_threaded`.

I mostly ask because the other fn is named `build`. Another option would be to rename `build` to be `multi_threaded`.

Thoughts?

/cc @sdroege @seanmonstar @olix0r.

Reply:
I would include the word "build" here, it allows to easily distinguish from the function name alone whether this just sets more properties on the builder or finished/builds the builder.

`build_single_threaded` sounds fine to me

Reply:
Done